content/guides/virtualizing/aws.md

```diff
@@ -0,0 +1,156 @@
+
+We're nearly done! All we need is to store our credentials in variables, since we will use them multiple times during the next section of the tutorial.
+
+For convenience, we will set our [AWS Security Credentials](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html) as two variables; `AWS_KEY` and `AWS_SEC`.
+
+```
+export AWS_KEY="<YOUR AWS KEY>"
+export AWS_SEC="<YOUR AWS SECRET>"
+```
+
+Once again, make sure to replace the two placeholders in between of the `<>`'s with your credentials.
+
+### Preparing the AMI <a name="part_ami">
+
+##### Uploading the disk image
+
+Now that you have gotten everything set and in order, we can now upload the image over to Amazon!
+
+```
+ec2-import-volume \
+    --format vhd \
+    --volume-size 4 \
+    --region us-east-1 \
+    --availability-zone us-east-1a \
+    --bucket haiku-folder \
+    --owner-akid $AWS_KEY --owner-sak $AWS_SEC \
+    --aws-access-key $AWS_KEY --aws-secret-key $AWS_SEC \
+    Haiku.vhd
+```
+
+Before proceeding any further, we will go over some of the parameters which you may find necessary to change.
+
+- `--volume-size` is the parameter that defines the amount of space that will be allocated for the virtualized environment. In this case, we chose the number `4` because it is the same one as the one we picked during the installation process in the [VirtualBox tutorial](https://www.haiku-os.org/guides/virtualizing/virtualbox/).
+
+- `--region` is the parameter that sets the server's region, which is independent from other regions. The `--availability-zone` parameter sets the availability zone-- they also hold a degree of independence, but they're a part of something that could be compared to an interlinked cluster. You can learn more about this topic in the [AWS User Guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html)
+
+- `--bucket` defines the name of your Amazon S3 Bucket. You can find out more about about buckets [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html).
+
+
+##### Creating the AMI
+
+Now that we have uploaded our image, we can now proceed with making a snapshot.
+
+```
+ec2-create-snapshot \
+   --aws-access-key $AWS_KEY \
+   --aws-secret-key $AWS_SEC \
+   --region us-east-1 \
+   haiku-folder
+```
+
+Snapshots are particularly useful in our case, because we can make an AMI out of it. 
+
+```
+ec2-register \
+   --name "Haiku R1/beta1 AMI" \
+   --aws-access-key $AWS_KEY \
+   --aws-secret-key $AWS_SEC \
+   --region us-east-1 \
+   --architecture x86_64 \
+   --root-device-name /dev/sda1 \
+   --virtualization-type hvm \
+   --snapshot haiku
+```
+
+### Deploying the AMI on the AWS platform <a name="part_deployment">
+
+After creating the AMI, it will appear in the `AMIs` section of the AWS Explorer. You can now launch an instance of the AMI by clicking on the `My AMIs` tab and selecting your newly made Haiku AMI.
```

Comment:
Again we <-> you. Maybe go through the guide again and change to "we/our" where appropriate.

```suggestion
After creating the AMI, it will appear in the `AMIs` section of the AWS Explorer. We can now launch an instance of the AMI by clicking on the `My AMIs` tab and selecting our newly made Haiku AMI.
```